src/Platforms/AbstractPlatform.php

```diff
@@ -1299,251 +1299,380 @@ public function getDateDiffExpression($date1, $date2)
     /**
      * Returns the SQL to add the number of given seconds to a date.
      *
-     * @param string $date
-     * @param int    $seconds
+     * @param string             $date
+     * @param int|numeric-string $seconds
```

Comment:
If the string is supposed to be numeric, and SQL expressions are not possible here, is there still a point to that API change? It seems less convenient to use strings here. What am I missing?

Reply:
It is needed for the API consumer to be able to pass strings instead of integers in order to be compatible with both DBAL 3 and 4.

See https://github.com/doctrine/orm/pull/9742 for more details.